src/main/java/seedu/address/model/feature/Feature.java

```diff
@@ -9,9 +9,10 @@
  */
 public class Feature {
 
-    public static final String MESSAGE_CONSTRAINTS =
-            "Please indicate one of the following features to view: 1) calendar, 2) attendance, "
-                    + "3) performance";
+    public static final String MESSAGE_CONSTRAINTS = "You have provided an invalid feature. "
+            + "Please indicate one of the following features "
```

Comment:
Consider joining these lines together - they don't need that many line breaks.

I think it worsens our Coveralls since it increases the number of lines that are not covered by tests 😢.

Reply:
right, thanks for the suggestion!